test/controllers/PenaltyDetailsController.test.ts

```diff
@@ -30,14 +30,14 @@ describe('PenaltyDetailsController', () => {
         });
 
         it('should return 200 when trying to access page with a session', async () => {
-            const penaltyDetails: Record<string, any> = {
+            const penaltyIdentifier: Record<string, any> = {
```

Comment:
There is ready to use `PenaltyIdentifier` type, much more readable than `Record<string, any>`